sleap/gui/learning/dialog.py

```diff
@@ -1038,11 +1046,34 @@ def _update_use_trained(self, check_state=0):
         else:
             use_trained = check_state == QtCore.Qt.CheckState.Checked
 
-        for form in self.form_widgets.values():
-            form.set_enabled(not use_trained)
+        # check if self._resume_training is None, because it might not be initialized yet
+        resume_training = self._resume_training and self._resume_training.isChecked()
+
+        if self._use_trained_model is not None:
+            for form in self.form_widgets.values():
+                form.set_enabled(
+                    (not self._use_trained_model.isChecked())
+                    or (
+                        self._use_trained_model.isChecked()
+                        and self._resume_training.isChecked()
+                    )
+                )
+
+            if self._use_trained_model.isChecked():
+                self._resume_training.setEnabled(True)
+
+                if resume_training:
+                    self.form_widgets["model"].set_enabled(False)
+            else:
+                # change the checkbox to unchecked
+                self._resume_training.setChecked(False)
+                self._resume_training.setEnabled(False)
+        else:
+            for form in self.form_widgets.values():
+                form.set_enabled(not use_trained)
```

Comment:
Do we need this third conditional case? Isn't this function only called when either `_require_trained == True` or `_use_trained_model` is checked or `_resume_training` is checked?